plugins/search-react/api-report.md

```diff
@@ -317,9 +319,14 @@ export const SearchResultListLayout: (
 // @public
 export type SearchResultListLayoutProps = ListProps & {
   resultItems?: SearchResult_2[];
-  renderResultItem?: (resultItem: SearchResult_2) => JSX.Element;
+  renderResultItem?: (
+    value: SearchResult_2,
+    index: number,
+    array: SearchResult_2[],
```

Comment:
Similar comment here as in the other PR here, too: https://github.com/backstage/backstage/pull/13945/files#r985478038

Reply:
The reason why we added the index and the array of results is to use the item position and also the results hightlight in our internal result rank computation when rendering each item.